Resources/Lang/zh_CN.json

```diff
@@ -7,7 +7,8 @@
   "Icon": "♥",
   "NameColor": "#ffc0cb",
   "kofi": "Ko-Fi",
-  "update": "一键更新",
+  "update": "Github一键更新",
```

Comment:
Please do not change anyother language files except for en_US.json using github PR. It will be rewritten by crowdin updates and may cause conflicts.

If you want to add a translation do it via crowdin

Reply:
I have already removed the changes